rust/nasl-interpreter/src/built_in_functions/string.rs

```diff
@@ -248,6 +248,39 @@ fn chomp(_: &str, _: &dyn Sink, register: &Register) -> Result<NaslValue, Functi
     }
 }
 
+/// NASL function to lookup position of a substring within a string
+///
+/// The first positional argument is the *string* to search through.
+/// The second positional argument is the *string* to search for.
+/// The optional third positional argument is an *int* containing an offset from where to start the search.
+fn stridx(_: &str, _: &dyn Sink, register: &Register) -> Result<NaslValue, FunctionError> {
+    let positional = resolve_positional_arguments(register);
+    let haystack = match positional.get(0) {
+        Some(NaslValue::String(x)) => x,
+        _ => {
+            return Err(FunctionError::new(
```

Comment:
Although it looks good to me it is incompatible because another PR that refactors the error handling got merged and therefore the creation of `FunctionError` got changed.

Do you prefer that I'll resolve it in your PR and create a commit or do you prefer to do it?

Reply:
Done